src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -85,4 +105,18 @@ void uct_rdmacm_cm_cqs_cleanup(uct_rdmacm_cm_t *cm);
 
 size_t uct_rdmacm_cm_get_max_conn_priv();
 
+ucs_status_t
+uct_rdmacm_cm_get_reserved_qpn_context(uct_rdmacm_cm_t *cm,
+                                       struct ibv_context *verbs,
+                                       struct uct_rdmacm_cm_reserved_qpn_context **ctx_p);
+
+ucs_status_t
+uct_rdmacm_cm_reserved_qpn_blk_alloc(uct_rdmacm_cm_t *cm,
+                                     struct ibv_context *verbs,
+                                     struct uct_rdmacm_cm_reserved_qpn_context *ctx,
+                                     struct uct_rdmacm_cm_reserved_qpn_blk **blk_p);
+
+ucs_status_t
+uct_rdmacm_cm_reserved_qpn_blk_dealloc(struct uct_rdmacm_cm_reserved_qpn_blk *blk);
```

Comment:
general comment - pls add `const` to input parameters

Reply:
@evgeny-leksikov Can't add const to verbs parameter for the same reason mentioned also as below:
https://github.com/openucx/ucx/pull/6604#discussion_r615701579